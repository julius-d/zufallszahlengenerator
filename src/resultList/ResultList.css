.Results {
  text-align: center;
  font-size: large;
}

.newest-result {
  font-size: xx-large;
}
