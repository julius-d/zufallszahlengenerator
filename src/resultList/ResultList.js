import React, {Component} from 'react';
import "./ResultList.css";


export default class ResultList extends Component {

  render() {
    const numberOfResults = this.props.randomNumbers.length;
    return (
      <div className="Results">
        {this.props.randomNumbers.map((randomNumber, i) =>
          <div key={numberOfResults - i}>
            <strong className={i === 0 ? "newest-result" : ""}>{randomNumber}</strong>
          </div>)}
      </div>
    );
  }

}
