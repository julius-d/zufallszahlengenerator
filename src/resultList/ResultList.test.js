import React from 'react';
import renderer from 'react-test-renderer';
import ResultList from "./ResultList";

describe("ResultList", () => {

  it('renders correctly', () => {
    const tree = renderer.create(
      <ResultList randomNumbers={[1337,42]}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
