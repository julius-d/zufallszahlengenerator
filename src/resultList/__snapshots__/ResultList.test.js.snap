// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultList renders correctly 1`] = `
<div
  className="Results"
>
  <div>
    <strong>
      1337
    </strong>
  </div>
  <div>
    <strong>
      42
    </strong>
  </div>
</div>
`;
